modules/terraform-zscc-iam-aws/main.tf

```diff
@@ -120,6 +120,36 @@ resource "aws_iam_role_policy_attachment" "cc_session_manager_attachment" {
 }
 
 
+################################################################################
+# Define AWS SQS/SNS Policy
+################################################################################
+data "aws_iam_policy_document" "cc_tags_policy_document" {
+  version = "2012-10-17"
+  statement {
+    sid    = "CCTags"
+    effect = "Allow"
+    actions = [
+      "sqs:CreateQueue",
+      "sns:Subscribe",
+    ]
+    resources = ["*"]
+  }
+}
+
+resource "aws_iam_policy" "cc_tags_policy" {
+  count       = var.byo_iam == false ? var.iam_count : 0

```

Comment:
It needs a flag to turn on/off. The workload tagging feature is not a must.

Reply:
Done - default is false